tests/unit/project/test_project_info.py

```diff
@@ -175,7 +175,7 @@ def test_tab_in_yaml(self):
         self.assertThat(
             raised.message,
             Equals(
-                "found a tab character that violate intendation on line 5, column 1"
+                "found a tab character that violate indentation on line 5, column 1"
```

Comment:
ha, the test failure is due to this change, the typo is in the actual product, feel free to fix it.

Reply:
That's weird the tests pass for me with this change? I'll investigate.

Reply:
Was due to changes in libyaml - committed fix